tests/unit/constraints/test_tabular.py

```diff
@@ -787,1789 +787,283 @@ def test_reverse_transform_non_string(self):
-        table_data = pd.DataFrame({
-            'a': [1, 2, 3],
-            'b': [4, 2, 2],
-            'c': [7, 8, 9]
-        })
-        out = instance.is_valid(table_data)
-
-        # Assert
-        expected_out = [False, True, True]
-        np.testing.assert_array_equal(expected_out, out)
+        # Run / Assert
+        with pytest.raises(ValueError):
+            Inequality._validate_inputs(
+                low_column_name='a', high_column_name=['b', 'c'], strict_boundaries=True
+            )
 
-    def test_is_valid_scalar_is_none_multi_column(self):
-        """Test the ``GreaterThan.is_valid`` method.
+    def test__validate_inputs_incorrect_strict_boundaries(self):
+        """Test the ``_validate_inputs`` method.
 
-        If scalar is none, and high is multi column, then
-        the values in that column should all be higher than
-        in the low column.
+        Ensure the method crashes when ``strict_boundaries`` is not a bool.
 
         Input:
-        - Table with values above and below low.
-        Output:
-        - True should be returned for the rows where the high
-        column is above low.
+        - a string
+        - a string
+        - a non-bool
+        Side effect:
+        - Raise ``ValueError`` because column names must be strings
         """
-        # Setup
-        instance = GreaterThan(low='b', high=['a', 'c'], strict=False)
-        table_data = pd.DataFrame({
-            'a': [1, 2, 3],
-            'b': [4, 2, 2],
-            'c': [7, 8, 9]
-        })
-
-        # Run
-        out = instance.is_valid(table_data)
-
-        # Assert
-        expected_out = [False, True, True]
-        np.testing.assert_array_equal(expected_out, out)
+        # Run / Assert
+        with pytest.raises(ValueError):
+            Inequality._validate_inputs(
+                low_column_name='a', high_column_name='b', strict_boundaries=None
+            )
 
-    def test_is_valid_high_is_datetime(self):
-        """Test the ``GreaterThan.is_valid`` method.
+    @patch('sdv.constraints.tabular.Inequality._validate_inputs')
+    def test___init___(self, mock_validate):
+        """Test the ``Inequality.__init__`` method.
 
-        If high is a datetime and low is a column,
-        the values in that column should all be lower than
-        ``instance._high``.
+        The passed arguments should be stored as attributes.
 
         Input:
-        - Table with values above and below `high`.
-        Output:
-        - True should be returned for the rows where the low
-        column is below `high`.
+        - low_column_name = 'a'
+        - high_column_name = 'b'
+        Side effects:
+        - instance._low_column_name = 'a'
+        - instance._high_column_name = 'b'
+        - instance._diff_column_name = 'a#b'
+        - instance._operator = np.greater_equal
+        - instance.rebuild_columns = 'b'
+        - instance._dtype = None
```

Comment:
This is basically just copying the code and doesn't really add anything. Instead can we describe why certain parameters are being set as certain values?